googlecdn.js

```diff
@@ -31,6 +31,26 @@ function isFunction(fn) {
   return typeof(fn) === 'function';
 }
 
+function getUrl(item, type, version) {
+  var urlType = item[type];
+  if (urlType) {
+    return (isFunction(urlType)) ? urlType(version) : urlType;
+  }
+  if (item.url) {
+    return (isFunction(item.url)) ? item.url(version) : item.url;
+  }
+}
+
+function getArray(types) {
```

Comment:
Could you check for an existing function/module for this? Seems more node-y to reuse something for this instead.